vendor: Update gostdlib to Go 1.13

bdarnell opened the pull request:
This updates the behavior of crlfmt to be like newer versions of
goimports. In particular, this improves the handling of the
opentracing-go imports and should stop the adding/removing of the
package alias.

Release note: None

bdarnell committed:
vendor: Update gostdlib to Go 1.13

This updates the behavior of crlfmt to be like newer versions of
goimports. In particular, this improves the handling of the
opentracing-go imports and should stop the adding/removing of the
package alias.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/42076)
<!-- Reviewable:end -->

bdarnell commented:
The sql.y change is because we were previously relying on goimports in the codegen pipeline to insert some missing imports. For some reason this no longer finds the `sql/types` import correctly, so I added it to the source file.

bdarnell committed:
*: Run new `gofmt -s`

Now insists on lowercase e in floating point exponent notation, which
is responsible for most of the changes. Also some whitespace changes
in a few files.

Release note: None

bobvawter reviewed (approved):
LGTM

> Reviewed 3 of 3 files at r1, 28 of 28 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/42076#-:-LsYsR6HDqU3AMsWuqo1:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

bdarnell commented:
bors r=bobvawter
